src/components/MultiHopTrade/hooks/useSupportedAssets.tsx

```diff
@@ -23,19 +24,20 @@ export const useSupportedAssets = () => {
   const [supportedBuyAssets, setSupportedBuyAssets] = useState<Asset[]>([])
   const [supportedBuyAssetsIds, setSupportedBuyAssetsIds] = useState<Set<AssetId>>(new Set())
 
+  const isSnapInstalled = useIsSnapInstalled()
   useEffect(() => {
     ;(async () => {
       const assetIds = await getSupportedSellAssetIds(enabledSwappers)
       const filteredAssetIds = new Set<AssetId>()
-      assetIds.forEach(assetId => {
+      assetIds.forEach(async assetId => {
         const chainId = fromAssetId(assetId).chainId
-        if (walletSupportsChain({ chainId, wallet })) {
+        if (await walletSupportsChain({ chainId, wallet, isSnapInstalled })) {
```

Comment:
do any of these need to be async? `walletSupportChain` is still sync.

Reply:
Leftover from previous hack, will be removed next diff 🙏🏽